client/src/containers/PageNotFound/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Button } from 'antd';
+import { Link } from 'react-router-dom';
+import * as ROUTES from '../../constants/routes';
+import notFoundImg from '../../assets/images/notFoundImg.jpg';
+import './style.css';
+
+const PageNotFound = () => {
+  return (
+    <div className="container">
```

Comment:
@AlaaSaadeddin It might be a good idea to give this a more descriptive class name, like `not-found-container`, because `container` might be a common class name that is reused in other components (and then this component could possibly get the other components' style rules)

Reply:
Yes , You are right :+1: 
thank you very much and welcome back :smiley:  @tomduggan85

Reply:
thank you!